azure-pipelines.yml

```diff
@@ -150,7 +147,6 @@ jobs:
 
     - job: Linux
       pool:
-        name: Hosted Ubuntu 1604
```

Comment:
I imagine this is breaking the Linux build?

Reply:
Shouldn't be. I'll try adding `vmImage: ubuntu-16.04`.

Reply:
Looks like that worked. Gogogog